ed448-goldilocks/src/decaf/points.rs

```diff
@@ -177,8 +177,11 @@ impl Group for DecafPoint {
         let mut bytes = DecafPointRepr::default();
 
         loop {
-            rng.try_fill_bytes(bytes.as_mut())?;
-            if let Some(point) = Self::from_bytes(&bytes).into() {
+            rng.try_fill_bytes(&mut bytes)?;
+            if let Some(point) = Self::from_bytes(&bytes)
+                .into_option()
+                .filter(|point| point != &Self::IDENTITY)
```

Comment:
The borrow here is necessary? I thought `PartialEq` borrowed its operands to begin with.

Reply:
Aah, `filter` borrows. Yeah, maybe `|&point|` instead?